dc-cudami-editor/src/components/lists/PagedIdentifierTypeList.jsx

```diff
@@ -40,13 +43,23 @@ const PagedIdentifierTypeList = ({apiContextPath = '/'}) => {
       <hr />
       <Card className="border-top-0">
         <CardBody>
-          <Pagination
-            changePage={({selected}) => setPageNumber(selected)}
-            numberOfPages={numberOfPages}
-            pageNumber={pageNumber}
-            totalElements={totalElements}
-            type={type}
-          />
+          <div className="d-flex justify-content-between">
+            <Pagination
+              changePage={({selected}) => setPageNumber(selected)}
+              numberOfPages={numberOfPages}
+              pageNumber={pageNumber}
+              totalElements={totalElements}
+              type={type}
+            />
+            {totalElements > 0 && (
+              <ListSearch
+                isHighlighted={totalElements === 0 && searchTerm}
```

Comment:
`totalElements === 0` ist hier doch immer `false` (s. Z. 54) :thinking:

Reply:
Vollkommen richtig, der Test in Z. 54 war Blödsinn :see_no_evil: